.pylintrc

```diff
@@ -0,0 +1,7 @@
+[MASTER]
+ignore =
+    typings
+
+[MESSAGES CONTROL]
+disable =
+    invalid-name
```

Comment:
seems a bit broad but if that's your intent, fine.

Reply:
Yeah, unfortunately pylint `invalid-name` doesn't allow for single character variables. It is true that as a rule single character variables should be avoided. But, when the variable is only used "hyper locally" in the space of a few lines, then these single characters are fine. Hence the disabling of that rule.

...but... after writing the above, I wanted to check to see if there was another, less general way, of achieving that. And, there is!

https://stackoverflow.com/a/35182977/3912576

I will remove `invalid-name` and make it explicit instead. Thank you :slightly_smiling_face:

Reply:
Fixed in 814c785